src/httpSpansAgent.test.js

```diff
@@ -38,7 +38,7 @@ describe('HttpSpansAgent', () => {
 
     const requests = AxiosMocker.getAxiosMocker().history.post;
     expect(requests.length).toEqual(1);
-    expect(requests[0].data).toEqual(reqBody);
+    expect(JSON.parse(requests[0].data)).toEqual(reqBody);
```

Comment:
@OrrLumigo how will affect the data arriving to our BE?

Reply:
this is just the Axios mocker its not the actual Axios lib